complete/_rg

```diff
@@ -182,6 +182,7 @@ _rg() {
     + '(pcre2)' # PCRE2 options
     {-P,--pcre2}'[enable matching with PCRE2]'
     $no'(pcre2-unicode)--no-pcre2[disable matching with PCRE2]'
+    '--pcre2-version[print the version of PCRE2 used by ripgrep, if available]'
```

Comment:
If i'm understanding correctly (that it has basically the same usage semantics as `--version`), this belongs under the `+ '(exclusive)'` group